contracts/AllDay.cdc

```diff
@@ -433,6 +438,8 @@ pub contract AllDay: NonFungibleToken {
         pub var maxMintSize: UInt64?
         // Updates each time we mint a new moment for the Edition to keep a running total
         pub var numMinted: UInt64
+        pub let deserialized: Bool?
+        pub let presetSerial: UInt64?
```

Comment:
How are you planning on getting around the upgrade restrictions that you can't add new fields to existing types?

Reply:
good point. @sadief i think what we can do is to modify the mint function signature to take in an optional Serial. If an edition is deserialized or not can be part of the edition offchain data